gooddata-java/src/main/java/com/gooddata/sdk/service/GoodData.java

```diff
@@ -235,6 +235,18 @@ protected GoodData(GoodDataEndpoint endpoint, Authentication authentication, Goo
         auditEventService = new AuditEventService(getRestTemplate(), accountService, settings);
         executeAfmService = new ExecuteAfmService(getRestTemplate(), settings);
         lcmService = new LcmService(getRestTemplate(), settings);
+
+        try {
+            Class.forName("com.github.sardine.Sardine", false, getClass().getClassLoader());
+            dataStoreService = new DataStoreService(getHttpClient(), getRestTemplate(), gdcService, endpoint.toUri());
+        } catch (ClassNotFoundException e) {
+            if (logger.isWarnEnabled()) {
+                logger.warn("Optional dependency Sardine not found - WebDAV related operations are not supported");
```

Comment:
What about to log only in info level? Warning is for me like something that I should fix, but this is gonna be intentional in most cases.

Reply:
yeah that sounds reasonable